custom-schematics/src/token-option/files/generic/preload-__name@dasherize__.ts

```diff
@@ -0,0 +1,16 @@
+import registerTokenOption from 'app/token-options/token-option-register-helper';
+import { <%= classify(name) %>TokenOption } from './<%= dasherize(name) %>-token-option';<% if (useCustomResolver) { %>
+import { <%= classify(name) %>TokenOptionResolver } from './<%= dasherize(name) %>-token-option-resolver';<% } %>
+import { <%= classify(name) %>TokenOptionFieldComponentFactory } from './<%= dasherize(name) %>-token-option-field-component-factory';<% if (useCustomRequest) { %>
+import { <%= classify(name) %>RequestResolver } from './<%= dasherize(name) %>-request-resolver';<% } %>
+import { <%= classify(name) %>RequestHandler } from './<%= dasherize(name) %>-request-handler';
+
+registerTokenOption(<%= classify(name) %>TokenOption, {
+    // order: 0,
```

Comment:
The `order` attribute does not influence anything in the Token ATM configuration. It is purely used to indicate the order that the token option type should be shown in the modal of creating new token option.

Since more and more token options are being added to Token ATM, I think a better idea would be making the single-selection field in the modal of creating new token option searchable (and we already have such a form field implemented). The `order` argument is thus intentionally made optional to serve as a temporary mitigation.